test/cmdlineTests.sh

```diff
@@ -45,15 +46,14 @@ case "$OSTYPE" in
         SOLC="$REPO_ROOT/${SOLIDITY_BUILD_DIR}/solc/solc"
         ;;
 esac
+echo "${SOLC}"
 
 INTERACTIVE=true
 if ! tty -s || [ "$CI" ]
 then
     INTERACTIVE=""
 fi
 
-FULLARGS="--optimize --ignore-missing --combined-json abi,asm,ast,bin,bin-runtime,compact-format,devdoc,hashes,interface,metadata,opcodes,srcmap,srcmap-runtime,userdoc"
-
 # extend stack size in case we run via ASAN
 if [[ -n "${CIRCLECI}" ]] || [[ -n "$CI" ]]; then
     ulimit -s 16384
```

Comment:
Wasn't the function `compileFull` below moved?

Reply:
It was moved, I guess I didn't corectly branch from original branch to my fork...
Now fixed